src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="box">
+        <legend>Personal information</legend>
+        <label class="form_field" for="surname">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          minlength="2"
+          maxlength="50"
+          id="surname">
```

Comment:
See checklist item 3 for correct indentations for a tags and attributes.
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            minlength="2"
            maxlength="50"
            id="surname"
          >
```